src/clusterfuzz/_internal/bot/tasks/setup.py

```diff
@@ -744,17 +744,15 @@ def _is_data_bundle_up_to_date(data_bundle, data_bundle_directory):
   return False
 
 
-def trusted_get_data_bundle_directory(fuzzer):
-  """For fuzz_task which doesn't get data bundles in an untrusted manner."""
-  # TODO(metzman): Delete this when fuzz_task is migrated.
-  # Check if we have a fuzzer-specific data bundle. Use it to calculate the
-  # data directory we will fetch our testcases from.
-  data_bundle = data_types.DataBundle.query(
-      data_types.DataBundle.name == fuzzer.data_bundle_name).get()
-  return get_data_bundle_directory(fuzzer, data_bundle)
+def get_data_bundle_directory(fuzzer, setup_input):
+  assert setup_input.data_bundle_corpuses[0].data_bundle
+  data_bundle = setup_input.data_bundle_corpuses[0].data_bundle
```

Comment:
Add a comment to explain why we are only doing [0] ?

Reply:
Done.